.github/workflows/cron-job-unit-tests.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        jdk: [ '11', '21' ]
+        jdk: [ '11', '21.0.4' ]
```

Comment:
We also need similar change in the other occurrences in this file, as the reporting jobs involve using the JDK as part of the artifact prefix for fetching the artifacts produced by this job.

Reply:
Done thanks for catching.